src/Storage/Entry.ts

```diff
@@ -3,6 +3,6 @@ export interface Entry {
   readonly ipfsFileHash: string
   readonly claimId?: string
   readonly lastDownloadAttempt?: number
-  readonly lastDownloadSuccess?: number
+  readonly downloadSuccessTime?: number
```

Comment:
I have a doubt here, I found `downloadSuccessTime` in several places with a `null` as a default value. Is it ok?

in this file https://github.com/poetapp/node/blob/de7c15ec963a8e07796a654e0c3627470edce167/src/Storage/ClaimController.ts

Reply:
Good catch. I did not think about that.

I think because the property is optional it is ok, we are using null to make sure the mongo database has the property, where if we passed undefined to mongo I am pretty sure mongo would ignore the property.

Given that this was pre-existing code that was accidentally removed at some point. I think this is something we should come back to after mainnet and maybe we can come up with better ways of working with mongo flows and data.